fossunited/foss_profiles/doctype/foss_user_profile/templates/foss_user_profile.html

```diff
@@ -380,7 +380,7 @@ <h4 class="mr-1">{{doc.full_name}}</h4>
                 'doctype': table,
             },
             callback: (r) => {
-                d = new frappe.ui.Dialog({
+/*                d = new frappe.ui.Dialog({
```

Comment:
Let's not keep commented our code. That's what source control is for.

Reply:
Sure, removing it